packages/react-reconciler/src/ReactFiberWorkLoop.new.js

```diff
@@ -1603,6 +1603,11 @@ export function isAlreadyRendering() {
   );
 }
 
+export function isAlreadyRenderingProd() {
+  // Used to throw if certain APIs are called from the wrong context.
```

Comment:
Nit: Rename either this function or the other function or both so it's less confusing which is which. When in doubt I prefer to make them more specific, e.g. `isInvalidExecutionContextForEventFunction`